draft-ietf-quic-applicability.md

```diff
@@ -274,6 +275,29 @@ might not be desirable. For such streams, QUIC could either provide an
 explicit interface to control prioritization, or derive the prioritization
 decision from the reliability level of the stream.
 
+# Port Selection
+
+As QUIC is a general purpose transport protocol, there are no requirements that
+servers use a particular UDP port for QUIC in general but rather the same port
```

Comment:
I'd put a sentence break after "...in general.  Instead, the same port number is used as...."